File: wellness-mentor-service/src/main/java/com/stackroute/wellnessmentorservice/model/BookedAppointment.java
package com.stackroute.wellnessmentorservice.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class BookedAppointment {
    private AppointmentSlot appointmentSlot;
    private String customerEmail;
    private String customerName;
}

File: wholesome-care-frontend/src/app/mentor-profile/BookedAppointment.ts
import { AppointmentSlot } from "./AppointmentSlot";

export class BookedAppointment{
    appointmentSlot:AppointmentSlot;
    customerEmail:String;
    customerName:String;
    // dateTime:Date;
    constructor(){
        this.appointmentSlot=new AppointmentSlot();
        this.customerEmail='';
        this.customerName='';
        // this.dateTime = new Date();
    }
}